RELEASE-CHECKLIST.md

```diff
@@ -0,0 +1,9 @@
+## Release Checklist
+
+- Run `generate.sh` to update man pages and shell-completion files.
```

Comment:
This needs to happen after the version is bumped in `Cargo.toml`, or the new version won't end up in the man page